public/app/features/plugins/sql/components/configuration/useMigrateDatabaseFields.test.ts

```diff
@@ -0,0 +1,71 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import { DataSourceSettings } from '@grafana/data';
+
+import { SQLConnectionDefaults } from '../../constants';
+import { SQLOptions } from '../../types';
+
+import { useMigrateDatabaseFields } from './useMigrateDatabaseFields';
+
+describe('Database Field Migration', () => {
+  let defaultProps = {
```

Comment:
Wish there was a better way to build out these props than building object literals by hand.

Reply:
Ya... I think we're stuck with this unfortunately 😦